cmd/nerdctl/system_prune_linux_test.go

```diff
@@ -32,6 +32,11 @@ import (
 
 func TestSystemPrune(t *testing.T) {
 	testutil.RequiresBuild(t)
+	// FIXME: for some reason, using a dedicated namespace does work well with rootless, but fails with rootful... :/
```

Comment:
No idea what's going on here, and why this happens only with rootful.
Will investigate later when I get to system tests...